migrations/challenges/sync-all-challenges.js

```diff
@@ -0,0 +1,40 @@
+import Bluebird from 'Bluebird';
+
+import { model as Challenges } from '../../website/server/models/challenge';
+import { model as User } from '../../website/server/models/user';
+
+async function syncChallengeToMembers (challenges) {
+  challenges.forEach(async function (challenge) {

```

Comment:
and while you have a limit(10) in the query it'll start a new one as soon as the forEach has run without waiting for the first 10 challenges to be processed